noxfile.py

```diff
@@ -41,7 +41,13 @@ def codespell(session: nox.Session) -> None:
 
 @poetry_session
 def mypy(session: nox.Session) -> None:
-    session.run("mypy", "crescent")
+    session.run("poetry", "run", "mypy", "crescent")
+
+
+@poetry_session
+def pyright(session: nox.Session) -> None:
+    session.run("poetry", "run", "pyright")
+    session.run("poetry", "run", "pyright", "--verifytypes", "crescent")
```

Comment:
Do we need to do this twice?

Reply:
Yes, first for type checking, second for verify types which checks the type completeness of crescent.

Reply:
Alright. Usually flags like that add/modify behavior, rather than do something else completely.

Reply:
Yeah it's a bit weird imo too